docs/_docs/versions.md

```diff
@@ -7,112 +7,161 @@ redirect_from: /docs/
 
-If you're working with an integration that's currently private, the _Promote_ option isn't used. Instead, make sure users you'd like to test or move to the new integration version are [invited](./testing#how-to-invite-others-to-test-new-integrations) to that version, and encourage them to try it out.
+To deprecate a Private version, select the **gear icon** next to the version number on the Versions page, then select **Deprecate**.
 
-## Deprecate an Older Version of Your Integration
+![The window that opens when selecting 'Deprecate'](https://cdn.zappy.app/7e62435659f494a9143aa1602640ef97.png)
 
-![Deprecate Zapier integration version](https://cdn.zapier.com/storage/photos/dd6acdd75278ecd733a5f3945ea641a2.png)
+> **Note:** You can also set a deprecation date using the CLI, with the `zapier deprecate` command. [More about that here](https://platform.zapier.com/cli_docs/docs#promoting-an-app-version).
 
-If you introduce breaking changes to your integration and want existing users to switch to the new version, you can deprecate the old version.
+When a deprecation date is set, Zapier sends a notification to that version’s users, to let them know about the need to upgrade to the latest Public version. Users can then manually update their Zaps, and ensure that the new version fits their workflows.
 
-Deprecation is only recommended if the older integration version will eventually stop working, such as if the related API will be removed. Zapier is normally a "set it and forget it" experience for users, so use this feature carefully.
+After the deprecation date has passed, the version will still be available in the Versions page for future reference. You can even invite users to it with the options in the Sharing page, like you would with a Private version.
 
-When you deprecate an integration version:
+### Should I deprecate or delete?
 
-* The Zaps that use it will continue to run until the deprecation date. On the deprecation date, they will be paused.
-* Zapier will send users with active Zaps a notification that they need to update their Zaps.
-* When adding a new Zap step, Zapier will no longer show the deprecated version of your integration as an option.
+You may have noticed that it’s also possible to _Delete_ versions from the Versions page. That option will fully remove the version from the Developer Platform, meaning that neither you nor users will be able to access it from that time — so use it with caution! 
 
-Zap steps using this integration version will include a `Deprecated` tag in the Zapier editor, to help prompt users to re-build the Zap with the new integration.
+In most cases, either deprecation or leaving a private version untouched are recommended, and deletion should only be used when absolutely necessary.
 
-To deprecate a version:
+## Can I start over?
```

Comment:
Maybe too complex for this section, but what do we think about adding something about `zapier link` here, for devs who started a new integration from scratch? Or maybe just have it listed as a note?

Reply:
I like that @nicolemichellethomas! I'll add something along those lines in the next commit.